### PR TITLE
Added Upload as Draft override

### PR DESCRIPTION
**Pull request by adelavina**

### CODE REVIEW

File: publish.go [hunk 1]
```diff
@@ -207,7 +208,10 @@ func createTrackRelease(whatsNewsDir string, versionCodes googleapi.Int64s, user
 }
 
 // releaseStatusFromConfig gets the release status from the config value of user fraction.
-func releaseStatusFromConfig(userFraction float64) string {
+func releaseStatusFromConfig(userFraction float64, uploadAsDraft bool) string {
+	if uploadAsDraft {
+		log.Infof("Release is a Draft release overriden by config.")
```
Comment (Szabadember) [inline publish.go:213]: Isn't there a `return releaseStatusDraft` statement missing?

File: publish_test.go [hunk 2]
```diff
@@ -20,7 +20,7 @@ func Test_releaseStatusFromConfig(t *testing.T) {
 	}
 	for _, tt := range tests {
 		t.Run(tt.name, func(t *testing.T) {
-			if got := releaseStatusFromConfig(tt.userFraction); got != tt.want {
+			if got := releaseStatusFromConfig(tt.userFraction, false); got != tt.want {
```
Comment (Szabadember) [inline publish_test.go:23]: `false` should not be hard wired there, we should test both cases

File: main.go [hunk 3]
```diff
@@ -78,7 +78,7 @@ func uploadApplications(configs Configs, service *androidpublisher.Service, appE
 func updateTracks(configs Configs, service *androidpublisher.Service, appEdit *androidpublisher.AppEdit, versionCodes []int64) error {
 	editsTracksService := androidpublisher.NewEditsTracksService(service)
 
-	newRelease, err := createTrackRelease(configs.WhatsnewsDir, versionCodes, configs.UserFraction)
+	newRelease, err := createTrackRelease(configs.WhatsnewsDir, versionCodes, configs.UserFraction, configs.uploadAsDraft)
```
Comment (Szabadember) [inline main.go:81]: There is a typo which will prevent this code from compiling:
`main.go:81:106: configs.uploadAsDraft undefined (type Configs has no field or method uploadAsDraft`

**Comment by adborbas** [comment]
Hi @adelavina,

Could you please let us know whether the PR is still relevant?

Thanks

**Comment by mateherber** [comment]
Hi @adelavina,

As a part of our effort to clean up our repositories we're trying to resolve our PRs. We're closing the PR as it seems stale. If this is still something you wish to contribute please reopen the PR utilizing our new guidelines and PR template.

**Pull request closed without merging**